src/main/kotlin/cz/startnet/utils/pgdiff/PgDiffOptions.kt

```diff
@@ -18,5 +18,27 @@ data class PgDiffOptions(
      * Whether to ignore whitespace while comparing content of functions.
      */
     val isIgnoreFunctionWhitespace: Boolean = false,
+
     val dropCascade: Boolean = false,
-)
\ No newline at end of file
+
+    val schemas: List<String> = emptyList(),
+
+    val excludeSchemas: List<String> = emptyList(),
+
+    val outputIgnoredStatements: Boolean = false
+) {
+
+    private val schemaInclPatterns = schemas.map(::Regex)
```

Comment:
:warning: this has slighly other behaviour than pg_dump:

this will not exclude the fe_sync and fe_dev schema
```
apgdiff -N fe -N ''fe_*''
```

whereas this will do
```
apgdiff -N fe -N ''fe_.*''
```

Reply:
jep i know, it is only regex

Reply:
i will change the metavar to make that clear